doc/README.md

```diff
@@ -19,52 +19,50 @@ to learn more about the format that is used.
 The documentation for the standard library is generated from
 the `.v` source files using coqdoc.
 
-Prerequisite
+Dependencies
 ------------
 
-To produce the complete documentation in HTML, the following tools are required:
+### HTML documentation
 
-  - Python 3
-  - Sphinx 1.6.5 (http://www.sphinx-doc.org/en/stable/)
-  - sphinx_rtd_theme
-  - pexpect
-  - beautifulsoup4
-  - Antlr4 runtime for Python 3
+To produce the complete documentation in HTML, you will need the dependencies
+of Coq listed in [`INSTALL`](/INSTALL). Additionally, the Sphinx-based
```

Comment:
"Coq dependencies" is more idiomatic but "dependencies of Coq" is quite understandable.

Reply:
> Or does "make sphinx" do the compile, too?

It does. And having an installed Coq is not sufficient.

Reply:
Isn't it true that if you're able to build Coq from the source tree, then you've satisfied the dependencies?  The way it's worded suggests that the reader should go read INSTALL, which may be unnecessary.  Maybe just add something like "(If you're able to build Coq, the dependencies are already satisfied.)"